src/monitor/pgautofailover--1.5--1.6.sql

```diff
@@ -254,15 +254,33 @@ DROP FUNCTION pgautofailover.remove_node(int);
 
 CREATE FUNCTION pgautofailover.remove_node
  (
-   node_id bigint
+   node_id bigint,
+   force   bool default 'false'
```

Comment:
Migration mostly looks good, did you check `\dx+ pgautofailover` differences?

Reply:
Yes, no diffs.